src/js/core.js

```diff
@@ -553,6 +554,27 @@ function MediumEditor(elements, options) {
         if (action === 'image') {
             return this.options.ownerDocument.execCommand('insertImage', false, this.options.contentWindow.getSelection());
         }
+        /* Issue: https://github.com/yabwe/medium-editor/issues/595
+         * If the action is to justify the text */
+        if (justifyAction.exec(action)) {
```

Comment:
Would you mind adding a test for this?  We will definitely want something to cover this code and make sure any other implementations still achieve the desired outcome.